### PR TITLE
doc: hot containers

### LINKED ISSUE #393: Add documentation on how to use hot containers

### PR DESCRIPTION
**Pull request by ccirello**
Fixes #393

### CODE REVIEW

File: docs/hot-containers.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+which we are going to get in the details here.
+
+The basic cycle comprises three steps: read standard input up to a previosly
+known point, process the work, the write the output to stdout with some
+information about when functions daemon should stop reading from stdout.
+
+In the case at hand, we start by plugging stdin to a HTTP request parser:
+
+```go
+r := bufio.NewReader(os.Stdin)
```
Comment (pedronasser) [inline docs/hot-containers.md]: Shouldn't have a loop somewhere here?

Comment (pedronasser) [inline docs/hot-containers.md]: Shouldn't have a loop somewhere here? Or the idea here really not show that part of the code?

File: docs/hot-containers.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+		"format": "http",
+		"max_concurrency": "1"
+	}
+}
+```
+
+`format` (mandatory) either "default" or "http". If "http", then it is a hot
+container.
+
+`max_concurrency` (optional) - the number of simultaneous hot containers for
```
Comment (pedronasser) [inline docs/hot-containers.md]: Should we inform that default max_concurrency is 1 by default?

Comment (ccirello) [inline docs/hot-containers.md]: Good catch.

**Comment by ccirello** [comment]
done @pedronasser

**Pull request merged**